src/panels/logbook/ha-panel-logbook.js

```diff
@@ -22,6 +23,11 @@ import LocalizeMixin from '../../mixins/localize-mixin.js';
  * @appliesMixin LocalizeMixin
  */
 class HaPanelLogbook extends LocalizeMixin(PolymerElement) {
+  constructor() {
+    super();
+    this.entityPicked = this._entityPicked.bind(this);
```

Comment:
This is not necessary with Polymer event listeners.

Reply:
Otherwise we lose context.

Reply:
Use `on-change` and you don't need it.